server/src/main/java/org/opensearch/search/profile/query/ConcurrentQueryProfileBreakdown.java

```diff
@@ -216,6 +221,12 @@ Map<Collector, Map<String, Long>> buildSliceLevelBreakdown() {
                         (key, value) -> (value == null) ? sliceLeafTimingTypeEndTime : Math.max(value, sliceLeafTimingTypeEndTime)
                     );
                 }
+                // Only when we've checked all leaves in a slice and still find no invocations, then we should set the slice start/end time
+                // to the default 0L.
```

Comment:
Lets add to the comment that `buildQueryBreakdownMap` expects `timingTypeSliceStartTimeKey` and `timingTypeSliceEndTimeKey` in the slice level breakdowns.

Reply:
It has been added now.